packages/blade/src/components/Table/TableFooter.web.tsx

```diff
@@ -107,16 +107,27 @@ const StyledFooterCell = styled(FooterCell)<{
 const _TableFooterCell = ({
   children,
   textAlign,
+  gridColumnStart,
+  gridColumnEnd,
+  gridRowStart,
+  gridRowEnd,
   ...rest
 }: TableFooterCellProps): React.ReactElement => {
   const isChildrenString = typeof children === 'string';
   const { backgroundColor, rowDensity } = useTableContext();
 
+  const hasRowSpan = Boolean(gridRowStart && gridRowEnd);
+  const gridRowValue = hasRowSpan ? `${gridRowStart} / ${gridRowEnd}` : undefined;
+
   return (
     <StyledFooterCell
+      className={hasRowSpan ? 'has-row-spanning' : ''}
+      gridColumnStart={gridColumnStart}
+      gridColumnEnd={gridColumnEnd}
       $backgroundColor={backgroundColor}
       $rowDensity={rowDensity}
       $textAlign={textAlign}
+      style={{ gridRow: gridRowValue }}
```

Comment:
```suggestion
      gridRow={gridRowValue}
```

we can add this prop on StyledFooterCell

Reply:
At first glance, I made the same change but later reverted it to style changes. I’ll bring back the original change.